zebra-utils/zcash-rpc-diff

```diff
@@ -16,6 +16,7 @@ function usage()
 
 # Override the commands used by this script using these environmental variables:
 ZCASH_CLI="${ZCASH_CLI:-zcash-cli}"
+ZCASH_RPC_COOKIE_FILE="${ZCASH_RPC_COOKIE_FILE:-.cookie}"
```

Comment:
I like this idea, but the current code always requires a cookie file, and it overrides any password authentication in the config. 

As an alternative, we could add a fully customisable set of arguments for each node?

That would look like:
```sh
ZEBRAD_EXTRA_ARGS="${ZEBRAD_EXTRA_ARGS:-}"
ZCASHD_EXTRA_ARGS="${ZCASHD_EXTRA_ARGS:-}"
…
$ZCASH_CLI $ZEBRAD_EXTRA_ARGS getinfo > "$ZEBRAD_RELEASE_INFO"
…
$ZCASH_CLI $ZCASHD_EXTRA_ARGS getinfo > "$ZCASHD_RELEASE_INFO"
…
```

And then you would call it like:
```sh
ZCASHD_EXTRA_ARGS="-rpccookiefile=your/cookie/file" zcash-rpc-diff getrawmempool
```

And for the password case:
```sh
ZCASHD_EXTRA_ARGS="-rpcpassword=your-password" zcash-rpc-diff getrawmempool
```

Reply:
Yea, this is ok. I added it at https://github.com/ZcashFoundation/zebra/pull/6035/commits/b94ed8f3fdce8508f3be826fad650be0ab077365 but only the zcashd extra arguments as they are not needed by now.